rest/context.go

```diff
@@ -72,6 +73,8 @@ type RequestContext interface {
 	// string is returned with the error set.
 	NextURL() (string, error)
 
+	BuildURL(string, string) (string, error)
```

Comment:
Yeah, named parameters would help... I omitted them because nothing else on the interface has them.